src/App.js

```diff
@@ -17,6 +32,8 @@ function App() {
         <span>Users: </span>
         {users.length}
       </p>
+
+      <TodoList prepared={preparedTodos} />
```

Comment:
prepared what? better just todos

```suggestion
      <TodoList todos={preparedTodos} />
```